Treat Ice.ClassGraphDepthMax < 1 as unlimited in Swift (#5813)
zeroc-ice/ice: changelog.d/swift/class-graph-depth-max-unlimited.md

- Fixed communicator initialization to treat `Ice.ClassGraphDepthMax` values less than 1 as unlimited, matching the

>   other language mappings. Previously, setting this property to 0 aborted the program with a `fatalError`.

Reviewer: ```suggestion
  other language mappings. Previously, setting this property to 0 or a negative value aborted the program.
```
